specs/lib/addPipelineExtrasSpec.js

```diff
@@ -8,6 +8,9 @@ describe('addPipelineExtras', function() {
 
     beforeAll(function(done) {
         fs.readFile(gltfExtrasPath, function(err, data) {
+            if (err) {

```

Comment:
Use `defined` here (though I know this will eventually go away when everything is promisified.